ui/src/Shared/services/products-listing-service.service.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
```

Comment:
Avoid repetition in the service name `products-listing-service.service.ts`

Reply:
Okay. Resolved the issue as suggested.